.github/workflows/test-and-deploy.yml

```diff
@@ -121,6 +97,71 @@ jobs:
         uses: sendgrid/dx-automator/actions/datadog-release-metric@main
         env:
           DD_API_KEY: ${{ secrets.DATADOG_API_KEY }}
+          
+  code-signing:
```

Comment:
What I understand is; we dont need a separate step to import the certificate now, where are we storing the certificate then?